api_tabular/utils.py

```diff
@@ -93,6 +106,14 @@
         "name": "{}__differs",
         "description": "Differs from in column: {} ({}__differs=value)",
     },
+    "isnull": {
```

Comment:
I think they show in swagger as parameters with values. I see a `allowEmptyValue` for aggregators for example.